components/RoleBindings.vue

```diff
@@ -79,7 +79,7 @@ export default {
     } else if ( this.roleScope === SCOPE_CLUSTER ) {
       hash.allRoles = this.$store.dispatch(`${ inStore }/findAll`, { type: RBAC.CLUSTER_ROLE });
     // } else if ( this.roleScope === SCOPE_GLOBAL ) {
-    // hash.allRoles = this.$store.dispatch('management/findAll', { type: RBAC.GLOBAL_ROLE });
+    // hash.allRoles = this.$store.dispatch('management/findAll', { type: MANAGEMENT.GLOBAL_ROLE });
```

Comment:
Looks like everything changed in this file was commented out code. Is this code still necessary? Or did you want to just update the types in case someone un-commented later?